### ISSUE TITLE
i18next-parser don't handle the interpolated defaultValue

### ISSUE THREAD

**Issue by jare-talend** [i18next/i18next-parser#88]
- Is this a bug or a feature request? bug
- What version are you running (`0.x` or `1.x`)? 1.x
- Which runner (cli, gulp...)? cli

## What I'm trying to do?
I want to parse a file with t called with a Interpolated defaultValue.

e.g.

src/test.js

```
t('KEY1', {
        defaultValue: '{{var}} translated',
});

t('KEY2', {
        defaultValue: ' translated',
});
```

## How am I doing it?

*Replace this section with your config file or cli command*

Here is my config file:

```
{
  output: 'locales'
}
```

The command I'm running:

```
i18next src/**/*.js -c ./i18n.json
```

## What was the result?
```
{
  "KEY2": " translated"
}
```
*Replace this with a description of the result you are getting*

## What was I expecting?

```
{
  "KEY1": "{{var}} translated"
  "KEY2": " translated"
}
```
the 0.x version works well

**Comment by karellm** [comment]
Thanks for the bug report, I will investigate this tonight.

**Comment by karellm** [comment]
This is fixed in `1.0.0-beta12`

**Comment by jare-talend** [comment]
thank you
